src/Quizzes/QuizList/QuizList.tsx

```diff
@@ -4,30 +4,46 @@ import { Link, useParams, useNavigate } from "react-router-dom";
 import { quizArray } from "../exampleQuizzes";
 import { Quiz } from "../types";
 import { createAvailabilityText } from "../utils";
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import * as client from "../client"
 import './index.css';
 
 export const QuizList = () => {
   const { courseId } = useParams();
   const navigate = useNavigate();
 
-  // TODO: Change this to use backend
-  // when you change this to use the backend and get all quizzes for a course, you'll need to then take the quizId and get the number of questions from the questions collection
-  const quizzesList: Quiz[] = quizArray.filter((quiz) => quiz.course === courseId);
-  const [quizzes, setQuizzes] = useState<Quiz[]>(quizzesList);
+  // determine if will need to get quizzes' questions/points too
+  const [quizzes, setQuizzes] = useState<Quiz[]>([]);
```

Comment:
IDT we need to, I changed it so that when we press save or save & publish on the quiz questions editor, it updates the quiz document in mongo, so a Quiz should already have it's points/questions in it